### DISCUSSION TITLE
有关间隔时间的问题

### DISCUSSION THREAD

**Discussion by wjw0611** [mikumifa/biliTickerBuy#399] [提问专区] [answered]
设置的1000ms间隔，但是程序运行中总是卡住，常常间隔时间达到2-5s甚至以上，电脑性能的问题吗（配置5600x+2060 双8g内存）还是网络的问题？有的群友和我一样不稳定不知道什么原因。
![QQ截图20240709185152](https://github.com/mikumifa/biliTickerBuy/assets/76196967/c245ac3f-2877-411d-86ed-fd576e179c00)
![QQ截图20240709185245](https://github.com/mikumifa/biliTickerBuy/assets/76196967/6b9e9d2b-02d3-4080-80b1-24ac171fcd6a)
有群友的间隔时间就很正常基本一直是1000ms附近，并且他也抢到了
![QQ图片20240709185839](https://github.com/mikumifa/biliTickerBuy/assets/76196967/2c9b7e8d-15a8-4310-bb15-0d17519124b7)

**Comment by SouthSeafireworks** [comment, marked as answer]
我现在也这样，刚开始的几天是好的，类似的情况也出现在抢第一波和第二波bw的时候，应该是脚本最近开始普及太多，服务器压力太大，导致这边的请求变慢了，这种情况还是推荐设小一点吧，设成默认的300s

**Comment by ZianTT** [comment]
请注意网络环境正常，尽量不要挂梯子等
